rgw: added missing documentation on "pubsub" in rgw_enable_apis (#39382)
ceph/ceph: doc/radosgw/notifications.rst

   S3 Bucket Notification Compatibility <s3-notification-compatibility>

> .. note:: In order to use bucket notifications make sure that notifications API is enabled. The rgw_enable_apis configuration should contain: "notifications".

Reviewer: ```suggestion
.. note:: To enable bucket notifications API, the `rgw_enable_apis` configuration parameter should contain: "notifications".
```
